contracts/LockedNORI.sol

```diff
@@ -565,41 +595,50 @@ contract LockedNORI is
     address to,
     uint256 atTime,
     uint256 amount
-  ) internal {
+  ) internal onlyRole(TOKEN_GRANTER_ROLE) {
```

Comment:
Isn't this redundant if it's enforced on the outer public function?

Reply:
Yep. Must've lost this fix in merging. Will fix